Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM registry.fedoraproject.org/fedora:30
```

Comment:
could you please document purpose of this image?

Reply:
Sure, but what do you think? I am completely lost. What would you like to write about it?
We can, of course, use 29.

Reply:
Tomas, neither I do understand your comment

Reply:
My point is: what is the purpose of the image this dockerfile is suppose to build: how and where is the image meant to be used.

Reply:
Look at the latest comment.